src/components/ui/popover/popover.component.tsx

```diff
@@ -222,18 +222,22 @@ export class Popover extends React.Component<PopoverProps, State> {
       <MeasureElement 
         shouldUseTopInsets={ModalService.getShouldUseTopInsets}
         onMeasure={this.onContentMeasure}>
-        {this.renderPopoverElement()}
+        <FalsyFC component={this.renderPopoverElement()} />
       </MeasureElement>
     );
   };
 
   public render(): React.ReactElement {
+    const renderAnchor = typeof this.props.anchor === 'function' 
+      ? this.props.anchor() 
+      : this.props.anchor
```

Comment:
Naaaah, can we get rid of this? The RenderProp does the same check actually. I believe there is a proper way to solve it